### PR TITLE
How to use the default cycles wallet - API

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
**Overview**
Basic description of the default wallet API for reference

### CODE REVIEW

File: modules/developers-guide/pages/default-wallet.adoc [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+[[wallet-authorize]]
+== Authorize a custodian
+
+Use the `authorize` method to authorize a principal as a custodian of a cycles wallet.
+
+For example:
+
+....
+dfx canister --network=alpha call f3yw6-7qaaa-aaaab-qaabq-cai authorize '(principal "zdm3q-uxmkz-lsghp-r737v-dgwav-rinn2-hs3zf-fnib3-2rylb-3kuek-hae")'
```
Comment (Eve832) [inline modules/developers-guide/pages/default-wallet.adoc:101]: This format worked for me - i.e. no quotes for the wallet ID, however, the Quick Start guide uses quotes and this did not work for me — it threw an error (3) message.

**Review by Eve832 — APPROVED** [review]

File: modules/developers-guide/pages/default-wallet.adoc [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+:company-id: DFINITY
+:sdk-short-name: DFINITY Canister SDK
+:sdk-long-name: DFINITY Canister Software Development Kit (SDK)
+
+As discussed in link:concepts/tokens-cycles{outfilesuffix}[Tokens and cycles], ICP tokens can be converted into *cycles* to power canister operations. 
+Cycles provide a largely stable value proposition and represent the operational cost of communication, computation, and storage that applications consume. 
+For example, cycles reflect the actual cost of physical hardware, rack space, energy, storage devices, and network bandwidth required by applications running on the {IC}.
+
+Unlike ICP tokens, cycles are only associated with canisters and not user or developer principals. 
+Instead, users and developers manage the distribution and ownership of cycles through a special type of canister called a *cycles wallet*.
```
Comment (p-shahi) [inline modules/developers-guide/pages/default-wallet.adoc:15]:
```suggestion
Unlike ICP tokens, cycles are only associated with canisters and not user or developer principals. Furthermore, certain actions such as creating new canisters require paying the associated create canister cycle fee.
Therefore, users and developers manage the distribution and ownership of cycles through a special type of canister called a *cycles wallet*.
```

File: modules/developers-guide/pages/default-wallet.adoc [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+For the purposes of local development, the {sdk-short-name} automatically creates a default cycles wallet for you in every project and most of the operations performed using the cycles wallet happen behind the scenes.
+For example, the cycles wallet acts on your behalf to register canister identifiers and deploy canisters on the local network.
+
+In a production environment, however, you often need to explicitly register and transfer cycles to new canisters, specify the principals that can act as custodians, and manage the principals with ownership rights. 
+You can perform these types of tasks using the methods in the default cycles wallet.
+
+You should keep in mind, however, that calls to the cycles wallet canister are executed using the cycles wallet identifier associated with the currently-active user identity.
```
Comment (p-shahi) [inline modules/developers-guide/pages/default-wallet.adoc:23]:
```suggestion
You should keep in mind, however, that calls to the cycles wallet canister are executed using the cycles wallet identifier associated with the currently-selected user identity.
```

Comment (p-shahi) [reply modules/developers-guide/pages/default-wallet.adoc:23]: nit: replace currently-active with currently-selected

**Review by p-shahi — APPROVED** [review]

**Pull request merged**